packages/components/src/components/popover/popover.jsx

```diff
@@ -67,7 +67,7 @@ const Popover = ({
                             ? is_open ?? ((is_hovered && message) || (is_bubble_hover_enabled && is_bubble_hovered))
                             : is_open ?? (is_hovered && message)
                     }
-                    position={alignment}
+                    positions={[].concat(alignment)}
```

Comment:
in the older version of `react-tiny-popover` we had `position` prop for `<Popover/>` component which  took `string` but now we have `positions` which accepts an array of strings . 
I could change all use cases and changed them to the new form which was not reasonable So instead I only convert the current position to an array.

Reply:
@farzin-deriv this fixed the issue that we had in DBot page.
@yashim-deriv as far as I checked there is no other issues regarding `react-tiny-popover` in DBot.